build.gradle

```diff
@@ -1026,6 +1037,12 @@ project(':share') {
     archivesName = "kafka-share"
   }
 
+  configurations {
+    mockitoAgent {
+      transitive = false
+    }
+  }
```

Comment:
Can we not do this once for all projects?

Reply:
but not all projects need `mockitoAgent`, eg `':test-common:test-common-runtime'`, `':group-coordinator:group-coordinator-api'`, so I think its better for configuring it for separate projects

Reply:
What's the downside of enabling it for those projects and how many such projects exist?

Reply:
The downside is these projects need to implement `libs.mockitoCore` dependency which it doesn't need, and the following projects doesn't need it `':test-common:test-common-runtime'`, `':group-coordinator:group-coordinator-api'`, `':examples'`, `':generator'`, `':tools:tools-api'`, `':shell'`, `':streams:streams-scala'`, `':streams:examples'`, `':connect:transforms'`, `':connect:json'`, `':connect:mirror-client'`,  `':connect:test-plugins'`

Reply:
Could we enable it automatically if the mockitCore dependency is added to a project?

Reply:
It's a good idea, I will try for it :)